components/vf-card-container/README.md

```diff
@@ -12,7 +12,8 @@ A multi-column container for `vf-card`.
 
 - Use one card container for all cards that belong to the same section.
 - Always include [a vf-section header](https://stable.visual-framework.dev/components/vf-section-header/index.html) to the card container. This ensures that your page will not have [skipped headings](https://www.w3.org/WAI/tutorials/page-structure/headings/#heading-ranks) between the cards and the preceding content outside the container.
-- If you do not want the section header to be visible, you can hide it using the class `vf-u-sr-only` (for screen reader only). See the [example with the hidden header below](#examples).
+-  If you do not want the section header to be visible, you can hide it by passing `hidden` as true (hidden: true) inside the `container_section__header` section in card container config.yml setting. This value (true) applies the class `vf-u-sr-only` (for screen reader only) to the section header and hides it. See the [example with the hidden header below](#examples).
+- Hiding the heading can only be applied to a section header that is not a link (i.e. [the default variant](https://stable.visual-framework.dev/components/vf-section-header/index.html#vf-section-header--default)). Section headers that act as links should not be hidden because when users navigate with the keyboard the header is selected as a link but it will not visible to them.
```

Comment:
@nikiforosk
"...as a link but it will not visible to them."

should read -

"...as a link but it will not be visible to them."

'be' is missing